common/_static/custom.css

```diff
@@ -1,27 +1,94 @@
+@font-face {
```

Comment:
That said, it should be hard-coded to "https://". We don't do scheme-less URLs these days.